### PR TITLE
[Internal review] Update script for script issue_2626

### PR DESCRIPTION
**Pull request by VjKlepikov**
ATF Test Scripts to check [12919](https://luxproject.luxoft.com/jira/browse/FORDTCN-12919)

This PR is **[ready]** for review.

### Summary
Move the expectation function for Lock Screen URL event after RegisterAppInterface response

### ATF version
develop

### CLA
- [x] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

File: test_scripts/Defects/8_0/3674_2_HMI_starts_PTU_before_UpdateAppList.lua [hunk 1]
```diff
@@ -153,6 +149,20 @@ local function registerAppWithPTU(pAppId, pMobConnId)
             { hmiLevel = "NONE", audioStreamingState = "NOT_AUDIBLE", systemContext = "MAIN" })
           session:ExpectNotification("OnPermissionsChange")
           :Times(AnyNumber())
+          local policyMode = SDL.buildOptions.extendedPolicy
+          if policyMode == policyModes.P or policyMode == policyModes.EP then
+            local lsEvent = common.run.createEvent()
+            common.getHMIConnection():ExpectEvent(lsEvent, "Lock Screen URL event")
+            session:ExpectNotification("OnSystemRequest")
+            :Do(function(_, d2)
+                if d2.payload.requestType == "LOCK_SCREEN_ICON_URL" then
```
Comment (GetmanetsIrina) [inline test_scripts/Defects/8_0/3674_2_HMI_starts_PTU_before_UpdateAppList.lua:158]: Maybe we need to add check for requestType in `policyTableUpdateHttp` function
`common.getMobileSession():ExpectNotification("OnSystemRequest")`

**Review by GetmanetsIrina — CHANGES_REQUESTED** [review]

File: test_scripts/Defects/8_0/3674_2_HMI_starts_PTU_before_UpdateAppList.lua [hunk 2]
```diff
@@ -93,9 +91,6 @@ local function policyTableUpdateProprietary()
             requestType = "PROPRIETARY" }, ptuFileName)
           common.getMobileSession(id):ExpectResponse(corIdSystemRequest, { success = true, resultCode = "SUCCESS" })
           :Do(function() os.remove(ptuFileName) end)
-        elseif d2.payload.requestType == "LOCK_SCREEN_ICON_URL" then
-          common.getHMIConnection():RaiseEvent(lsEvent, "Lock Screen URL event")
-        end
       end)
     :Times(AtMost(2))
```
[the remark this answers is not in the document]
Comment (VjKlepikov) [reply test_scripts/Defects/8_0/3674_2_HMI_starts_PTU_before_UpdateAppList.lua]: done in [0b249a18](https://github.com/LuxoftSDL/sdl_atf_test_scripts/pull/158/commits/0b249a181c3457868d25f52c0dc2e567860dd2bb)

File: test_scripts/Defects/8_0/3674_2_HMI_starts_PTU_before_UpdateAppList.lua (hunk 1 above)
Comment (VjKlepikov) [reply test_scripts/Defects/8_0/3674_2_HMI_starts_PTU_before_UpdateAppList.lua:157]: I believe that is a redundant check for the mentioned issue

**Comment by VjKlepikov** [comment]
move to open, merged

**Pull request closed without merging**